{
	"document_title": "Security update for python-dulwich",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE-SU-2018:2047-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2018-07-24T07:20:39Z",
				"description": "current"
			}
		},
		"initial_release_date": "2018-07-24T07:20:39Z",
		"current_release_date": "2018-07-24T07:20:39Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "Security update for python-dulwich",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "This update for python-dulwich to version 0.18.5 fixes this security issue:\n\n- CVE-2017-16228: Dulwich, when an SSH subprocess is used, allowed remote\n  attackers to execute arbitrary commands via an ssh URL with an initial dash\n  character in the hostname (bsc#1066430).\n\nFor detailed changes please see https://www.dulwich.io/code/dulwich/\n  ",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "SUSE-SLE-Module-Public-Cloud-12-2018-1382",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/update/announcement/2018/suse-su-20182047-1/",
			"description": "Link for SUSE-SU-2018:2047-1"
		},
		{
			"type": "Self",
			"url": "https://lists.suse.com/pipermail/sle-security-updates/2018-July/004303.html",
			"description": "E-Mail link for SUSE-SU-2018:2047-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1066430",
			"description": "SUSE Bug 1066430"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2017-16228/",
			"description": "SUSE CVE CVE-2017-16228 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "SUSE Linux Enterprise Module for Public Cloud 12",
				"branch": [
					{
						"type": "Product Name",
						"name": "SUSE Linux Enterprise Module for Public Cloud 12",
						"full_product_name": {
							"text": "SUSE Linux Enterprise Module for Public Cloud 12",
							"product_id": "SUSE Linux Enterprise Module for Public Cloud 12",
							"cpe": "cpe:/o:suse:sle-module-public-cloud:12"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python-dulwich-0.18.5-4.3.1",
				"full_product_name": {
					"text": "python-dulwich-0.18.5-4.3.1",
					"product_id": "python-dulwich-0.18.5-4.3.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python-dulwich-0.18.5-4.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE Linux Enterprise Module for Public Cloud 12"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "Dulwich before 0.18.5, when an SSH subprocess is used, allows remote attackers to execute arbitrary commands via an ssh URL with an initial dash character in the hostname, a related issue to CVE-2017-9800, CVE-2017-12836, CVE-2017-12976, CVE-2017-1000116, and CVE-2017-1000117.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2017-16228",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"SUSE Linux Enterprise Module for Public Cloud 12:python-dulwich-0.18.5-4.3.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "10",
					"vector_v_2": "AV:N/AC:L/Au:N/C:C/I:C/A:C"
				},
				"score_set_v_3": {
					"base_score_v_3": "9.8",
					"vector_v_3": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					},
					"url": "https://www.suse.com/support/update/announcement/2018/suse-su-20182047-1/"
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2017-16228.html",
					"description": "CVE-2017-16228"
				},
				{
					"url": "https://bugzilla.suse.com/1053364",
					"description": "SUSE Bug 1053364"
				},
				{
					"url": "https://bugzilla.suse.com/1066430",
					"description": "SUSE Bug 1066430"
				},
				{
					"url": "https://bugzilla.suse.com/1071709",
					"description": "SUSE Bug 1071709"
				}
			]
		}
	]
}
